packages/bindings/src/oboe/oboe_api.cc

```diff
@@ -0,0 +1,103 @@
+#include "oboe_api.hh"
+#include "oboe_api.h"
+
+JsOboeAPI::JsOboeAPI(sw::CallbackInfo const info)
+    : sw::Class<JsOboeAPI, OboeAPI>(new OboeAPI(), info) {}
+Napi::Object JsOboeAPI::init(Napi::Env env, Napi::Object exports) {
+    return define_class("OboeAPI")
+        .method<&JsOboeAPI::getTracingDecision>("getTracingDecision")
+        .method<&JsOboeAPI::consumeRequestCount>("consumeRequestCount")
+        .method<&JsOboeAPI::consumeTokenBucketExhaustionCount>("consumeTokenBucketExhaustionCount")
+        .method<&JsOboeAPI::consumeTraceCount>("consumeTraceCount")
+        .method<&JsOboeAPI::consumeSampleCount>("consumeSampleCount")
+        .method<&JsOboeAPI::consumeThroughIgnoredCount>("consumeThroughIgnoredCount")
```

Comment:
It is okay for this PR but I want to give you a head up that `consumeThroughIgnoredCount` function is removed from liboboe. 
However, I am unable to build the staging library due the the EMU migration. 
If you like, you can remove it first so that you don't need to create another PR for that.